tests/unit/EmojiTest.js

```diff
@@ -78,14 +78,19 @@ describe('EmojiTest', () => {
         expect(EmojiUtils.containsOnlyEmojis('😄  👋')).toBe(true);
     });
 
-    it('replaces an emoji code with an emoji and a space', () => {
+    it('replaces an emoji code with an emoji and a space if addSpace is passed as true', () => {
         const text = 'Hi :smile:';
-        expect(EmojiUtils.replaceEmojis(text)).toBe('Hi 😄 ');
+        expect(EmojiUtils.replaceEmojis(text, true)).toBe('Hi 😄 ');
     });
 
-    it('will not add a space after the last emoji if there is text after it', () => {
+    it('will not add a space after the last emoji when there is text after it even if addSpace is passed as true', () => {
```

Comment:
```suggestion
    it('will not add a space after the last emoji when there is text after it', () => {
```
We can keep this one as is. The purpose here is to test that we're not adding a space if there's text after the emoji, regardless of which device we're on.